.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
+BasedOnStyle:  LLVM
```

Comment:
I think the `Google` style seems sane, haven't reviewed all of them yet though.

Reply:
Google also seems to have larger adoption